### ISSUE TITLE
After update of Docker Desktop (4.27.0), cant scan any local image anymore

### ISSUE THREAD

**Issue by K2ouMais** [docker/scout-cli#90]
Hello everyone,

since I updated Docker Desktop to the latest version, seems that something on Docker Scout got broken, while scanning local Images.

![image](https://github.com/docker/scout-cli/assets/52510455/73dbbbfd-4a96-465c-989a-90a0e7d838a0)

Here are my versions:
![image](https://github.com/docker/scout-cli/assets/52510455/e8a69c04-06f3-44d1-b1f8-48b855039d87)
![image](https://github.com/docker/scout-cli/assets/52510455/d4d57854-05e4-4cfd-a650-5b2500b5d0f5)

I am using WSL2 updated to the latest pre-release.

EDIT:
In Windows Powershell it seems to work:
![image](https://github.com/docker/scout-cli/assets/52510455/88cead45-b83f-47c9-8ef6-579219093ff5)

**Comment by cdupuis** [comment]
@K2ouMais thanks for reporting this. We’ll take a look immediately and report back. Sorry for the inconvenience.

**Comment by rainchei** [comment]
we got the same error too, here's our docker version

```
Client: Docker Engine - Community
 Version:           25.0.1
 API version:       1.44
 Go version:        go1.21.6
 Git commit:        29cf629
 Built:             Tue Jan 23 23:09:46 2024
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          25.0.1
  API version:      1.44 (minimum version 1.24)
  Go version:       go1.21.6
  Git commit:       71fa3ab
  Built:            Tue Jan 23 23:09:46 2024
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          1.6.27
  GitCommit:        a1496014c916f9e62104b33d1bb5bd03b0858e59
 runc:
  Version:          1.1.11
  GitCommit:        v1.1.11-0-g4bccb38
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

**Comment by cdupuis** [comment]
Could you please test this again but set `DOCKER_SCOUT_CACHE_FORMAT=tar`?

**Comment by K2ouMais** [comment]
This seems to work.

![image](https://github.com/docker/scout-cli/assets/52510455/408295cf-22ea-4fa1-ad24-605bead4966c)

**Comment by thaJeztah** [comment]
@cdupuis looks like possibly related to https://github.com/moby/moby/pull/44598. TL;DR the path to the blobs should be read from the manifest that's exported (which is now `blobs/<algorithm>`)

```bash
cat manifest.json | jq .
[
  {
    "Config": "blobs/sha256/3e4fd538a9a0b729be05707cf805388be2fb701cfd5d44c6542f1988e8aef6e3",
    "RepoTags": [
      "busybox:latest"
    ],
    "Layers": [
      "blobs/sha256/c2bf9493c1bf786e95e3eac7d406c20aa1b8a2d40916756e891627e9e8f8d119"
    ]
  }
]
```

**Comment by cdupuis** [comment]
@thaJeztah why would this only surface on WSL2?

**Comment by cdupuis** [comment]
This should be fixed in recent v1.4.1.
